problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,60 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  if (!this.stack1.length && !this.stack2.length) {
+    return null;
+  }
+
+  if (!this.stack2.length) {
+    while (this.stack1.length) {
+      this.stack2.push(this.stack1.pop());
+    }
+  }
+
+  return this.stack2.pop();
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.peek = function () {
+  if (this.stack2.length) {
+    return this.stack2[this.stack2.length - 1];
+  } else {
+    return this.stack1[0];
+  }
+};
+
+/**
+ * @return {boolean}
+ */
+MyQueue.prototype.empty = function () {
+  if (!this.stack1.length && !this.stack2.length) {
+    return true;
+  }
+
+  return false;
```

Comment:
```suggestion
  return !this.stack1.length && !this.stack2.length;
```

boolean인 값을 조건문을 거쳐 return하는 것보다, 바로 return해주는 게 더 간단한 방식인 것 같습니다~